popup.js

```diff
@@ -1,9 +1,57 @@
 function getHighlightedText(callback) {
-  chrome.tabs.executeScript( {
-    code: "window.getSelection().toString();"
+  // chrome.tabs.executeScript({
```

Comment:
actually, we don't need to worry about popup.js/popup.html. they are coming out anyway.